splipy/utils/nutils.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import division
+
+__doc__ = 'Implementation of convenience methods with respect to nutils integration.'
+
+from splipy import Curve, Surface, Volume
+import numpy as np
+
+
+def splipy_to_nutils(spline):
+    """ Return controlpoints according to nutils ordering """
+    n = len(spline)
+    dim = spline.dimension
+    if(type(spline) == Curve):
```

Comment:
Probably better: `isinstance(spline, Curve)`

Also, parens around if-conditions? What is this, C? :smile:

Reply:
That is better yes. Will fix

Reply:
That is better, yes. Will fix it.